tests/integrationv2/test_hello_retry_requests.py

```diff
@@ -138,3 +138,55 @@ def test_hrr_with_s2n_as_server(managed_process, cipher, provider, curve, protoc
     assert client_hello_count == 2
     assert server_hello_count == 2
     assert finished_count == 2
+
+# Default Keyshare for TLS v1.3 is x25519 
+TEST_CURVES = ALL_TEST_CURVES[1:] 
+@pytest.mark.uncollect_if(func=invalid_test_parameters)
+@pytest.mark.parametrize("cipher", TLS13_CIPHERS, ids=get_parameter_name)
+@pytest.mark.parametrize("provider", [OpenSSL])
+@pytest.mark.parametrize("curve", TEST_CURVES, ids=get_parameter_name)
+@pytest.mark.parametrize("protocol", [Protocols.TLS13], ids=get_parameter_name)
+@pytest.mark.parametrize("certificate", ALL_TEST_CERTS, ids=get_parameter_name)
+def test_hrr_with_default_keyshare(managed_process, cipher, provider, curve, protocol, certificate):
```

Comment:
This tests HRR with a single keyshare sent as the default (for TLS1.3 it is x25519). In this test we force into HRR mode by not sending x25519 as the serve's curve (see Line 142: ```
TEST_CURVES = ALL_TEST_CURVES[1:] ```). It isn't duplicating our basic tests as this is for integration testing with only one default curve. Once we remove support for multiple keyshares, this will be the only test.